lib/GWPARAM.ml

```
@@ -330,8 +549,17 @@ let init bname =
     images_d := Legacy.images_d)
 
 let init_etc bname =
-  if !init_done.status && bname = !init_done.bname then ()
+  if
+    (!init_done.status && bname = !init_done.bname)
+    || not (Sys.file_exists (!bpath bname))
```

You silently ignore this error, is it what you really want?